src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -6,9 +6,10 @@
 current_dir = os.path.dirname(os.path.realpath(__file__))
 configurations_dir = os.path.join(current_dir, 'configurations')
 cutters_dir = os.path.join(current_dir, 'cutters')
+temp_dir = os.path.join(current_dir, 'temp')
 
-TEMP_CONFIG_PATH = os.path.join(current_dir, 'temp', 'temp_config')
-DEBUG_MODE = False
+TEMP_CONFIG_PATH = os.path.join(temp_dir, 'temp_config')
+DEBUG_MODE = True
```

Comment:
Set this to false again. Don't think we need the debug output in master.

Reply:
Good spot, thanks!